src/overlays/actors/ovl_En_Niw_Lady/z_en_niw_lady.c

```diff
@@ -20,43 +36,535 @@ const ActorInit En_Niw_Lady_InitVars = {
     (ActorFunc)EnNiwLady_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Lady/EnNiwLady_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Lady/EnNiwLady_Destroy.s")
+static s16 EnNiwLadyMissingNiwTextIDs[] = {
+    0x5036, 0x5070, 0x5072, 0x5037, 0x5038, 0x5039, 0x503A, 0x503B, 0x503D, 0x503C,
+};
```

Comment:
this needs an `s` prefix. and I would maybe name it like, `sMissingCuccoTextIds`